tests/Aspire.Workload.Tests/Aspire.Workload.Tests.csproj

```diff
@@ -0,0 +1,59 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+
+    <RunTestsOnHelix>true</RunTestsOnHelix>
+    <TestUsingWorkloads>true</TestUsingWorkloads>
+    <InstallWorkloadForTesting>$(TestUsingWorkloads)</InstallWorkloadForTesting>
```

Comment:
This is kind of weird. Should this just be the default if `TestusingWorkloads` is true?

Reply:
Sure. It just shows the dependence between the two properties. In EndToEnd.Tests `TestUsingWorkloads` is `false` on CI/windows.